svc-bip-api/src/main/java/gov/va/vro/bip/config/RabbitMqConfig.java

```diff
@@ -74,23 +69,6 @@ public void configureRabbitListeners(RabbitListenerEndpointRegistrar registrar)
 
   @Bean
   DirectExchange bipApiExchange() {
-    return new DirectExchange(props.getExchangeName(), true, true);
-  }
-
-  @Bean
-  FanoutExchange deadLetterExchange() {
```

Comment:
Are we declaring the dead letter queue elsewhere now? It makes sense to remove this code since bip doesn't interact with it directly, just wondering where it's going to come from now.

Reply:
The dead letter exchange will be set via policy, and as soon as we develop a service to pull messages from the dead letter queue, that service will actively declare the exchange and the queue.